tests/test_api.py

```diff
@@ -773,9 +773,9 @@ def test_generate_manifest_file_based_annotations(
 
```

Comment:
Also, since we now have "integration" folder for all the integration test, should we move this file there too?

Reply:
There are a lot of interconnected pieces with these tests. I would need to spend some time to pull out specific components from this file and move everything over. I think it would be better to have some tech debt items that focus on moving items over.

I do agree though that (most) new tests should be created in the respective directories.